#153741764 CZARs should be able to draw cards by clicking

SEUNAGBEYE opened the pull request:
### What does this PR do?

- This allows czar to draw a card before the game begins and when a new round starts

### Description of Task to be completed?

- Modal containing cards a czar can draw from should be popped up before the game starts, it should also do so for every game rounds
- Card should be flipped on clicked and questions should be reveal

### How should this be manually tested?

- git clone this repo @ https://github.com/andela/blade-runner-cfh.git
- move into the blade-runner-cfh directory
- run `npm install` to install all dependencies
- go to `localhost:port/app` and click on the start game button then the modal pops for the czar to draw a card before the game begins



### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- #153741764

### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-01-17 at 12 59 35 am" src="https://user-images.githubusercontent.com/25517388/35019226-022bbeae-fb26-11e7-9d33-6a439ac80f70.png">

<img width="1440" alt="screen shot 2018-01-18 at 9 20 19 am" src="https://user-images.githubusercontent.com/25517388/35093550-f702782e-fc42-11e7-98fb-193d051a5b8c.png">

<img width="1440" alt="screen shot 2018-01-19 at 11 26 20 am" src="https://user-images.githubusercontent.com/25517388/35146560-b55374f4-fd0b-11e7-8095-22bf66bf46e0.png">



### 
Questions:

N/A

chinezenwosu commented on config/socket/game.js:
```diff
@@ -66,7 +67,7 @@ Game.prototype.payload = function() {
   return {
     gameID: this.gameID,
-    players: players,
+    players,
```
Take this up as the first item

chinezenwosu commented on config/socket/game.js:
```diff
@@ -331,49 +355,49 @@ Game.prototype.pickCards = function(thisCardArray, thisPlayer) {
     return this.players[playerIndex];
-  } else {
+  } 
     return {};
```
Please align this properly

chinezenwosu commented on config/socket/socket.js:
```diff
@@ -9,8 +9,7 @@ var avatars = require(__dirname + '/../../app/controllers/avatars.js').all();
 module.exports = function(io) {
-
-  var game;
+  
```
Remove extra space

Commit:
bug(chrome-fix): fix chrome bug

[Starts #154441264]

SEUNAGBEYE committed:
feature(czar-can-draw-card): czar can draw card

- czar can draw a card on the new game screen on game start
- czar can draw a card when a new round start
- when card are clicked they flip out

[Finishes #153741764]

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors

SEUNAGBEYE committed:
fix hound-ci errors
